bin/meshroom_photogrammetry

```diff
@@ -1,27 +1,35 @@
 #!/usr/bin/env python
 import argparse
+import os
 
 import meshroom
 meshroom.setupEnvironment()
 
 import meshroom.core.graph
-from meshroom.nodes.aliceVision.CameraInit import readSfMData
 from meshroom import multiview
 
 parser = argparse.ArgumentParser(description='Launch the full photogrammetry pipeline.')
-parser.add_argument('--input', metavar='FOLDER', type=str,
+parser.add_argument('--input', metavar='FOLDER_OR_SFM', type=str,
                     default='',
-                    help='Input folder or json file.')
+                    help='Input folder with images or SfM file (.sfm, .json).')
```

Comment:
help='Input folder containing images or file (.sfm or .json) with images paths and optionally predefined camera intrinsics.'